src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx

```diff
@@ -21,7 +21,7 @@ redirects:
   To use New Relic One and the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
-![Explorer in New Relic One](./images/NR1-explorer-intro.png "NR1-explorer-intro.png")
+![Explorer in New Relic One](./images/nrone20210722.png "NR1-explorer-intro.png")
```

Comment:
style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed